src/nplinker/pairedomics/podp_antismash_downloader.py

```
@@ -214,7 +213,7 @@ def podp_download_and_extract_antismash_data(
     GenomeStatus.to_json(gs_dict, gs_file)
 
     if missing == len(genome_records):
-        logger.warning('Failed to successfully retrieve ANY genome data!')
+        raise ValueError("No antiSMASH data found for any genome")
```

Are we sure we want to raise an error in the `get` method of the `PODPDownloader` if there is no antismash data?

---

If there is no antismash data, there will be no BGC data. So it makes sense to raise an error to stop execution.